website_product_attribute_link/data/product_group_data.xml

```diff
@@ -1,8 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
-<odoo>
-    <data noupdate="1">
-        <record id="base.group_user" model="res.groups">
-            <field eval="[(4, ref('product.group_product_variant'))]" name="implied_ids"/>
-        </record>
-    </data>
-</odoo>
\ No newline at end of file
+<odoo noupdate="1">
```

Comment:
Hi @ValentinoDefelice this field should be in security folder, becase it's modify group registry. furthermore change the name of the xml file